test/Microsoft.AspNet.Mvc.FunctionalTests/MvcSampleTests.cs

```diff
@@ -51,6 +52,37 @@ public class MvcSampleTests
             }
         }
 
+        [Theory]
```

Comment:
less important but confirm the new Index action also returns successfully.  don't need to confirm exact content.